enterprise/internal/batches/state/state.go

```diff
@@ -635,9 +635,9 @@ func computeDiffStat(ctx context.Context, db database.DB, c *btypes.Changeset, r
 		}
 
 		fs := file.Stat()
-		stat.Added += fs.Added
-		stat.Changed += fs.Changed
-		stat.Deleted += fs.Deleted
+
+		stat.Added += fs.Added + fs.Changed
+		stat.Deleted += (fs.Deleted + fs.Changed)
```

Comment:
Maybe also drop the parens here?

```suggestion
		stat.Deleted += fs.Deleted + fs.Changed
```